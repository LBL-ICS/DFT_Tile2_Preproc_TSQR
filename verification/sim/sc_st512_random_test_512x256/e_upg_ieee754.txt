BF75D9EF
BE9A8A28
3FAE641C
BF9D356F
BEB19E21
3F273B43
C0151F94
3F85E382
BF99E104
3ECD024C
3E013C22
3EBB15E5
3FC132FA
BF414A5E
3FC18196
BF129154
BF83D9C8
BF4B06B4
3F92768D
BA40E7CE
3F194BC4
4020ED43
3FB80EF2
BFBFC377
BF035D0E
BFB89DE1
BFA2E6DA
3E1E0DB6
BF381E7D
3F3C0F73
BE652C01
BE3C17BF
3EBDDDCF
3F0D688D
BF0E97DB
BEFDBA57
3ECD2AAF
BF4B725D
3E6E9EBB
BF4768A3
BE18C22E
BF0A5291
BE8CEEA8
3F98B407
BC112E47
3EB59C7E
3F7CA4A5
3F827065
BEBB2E98
BF837169
BFAC7228
C057795A
3F28DA40
3FB66CF4
3FBAF14F
3F3805B1
3E81C372
BF443012
3F8D3E79
3FB5B4E8
BFCFC6D1
3FA9465C
C0004107
3F04F0F4
3F31AFD1
BF1CD761
3DF404BF
3DA13DEF
BEE718D9
BF89CAEA
BE07E4FD
BEC92CC3
BFF99010
3FD52C1B
3E63B8AF
3E226292
3F697F68
BF2AF973
3F0159F4
BF26279A
BF966FF4
BF101BBD
BFB773E8
C005BAFA
3FBBE66C
3E0177F8
3F2538E9
3F6EC4C1
3F2BB868
BF4605F8
3FB27B8E
3F5E7FDC
3EDFC3B3
3F6F4813
BFBBAA19
BDE8AEEB
3ED3A849
3FD63DA5
3F23B04A
3ED346C5
BF66BCA7
3EBF2ED9
BFC73F9B
BDA19587
3F52C8D8
3F0966AC
3F4EA555
BF7E113C
BEE63F0E
3DFF5D81
3FD90BA3
3EF7A48F
BF08C871
3FEF4BFD
3F24D392
3FD92668
3C265288
3F0E2073
BFE40FCB
BF902BAD
3F127D7D
3EE57A92
3F4C922F
3F8241D5
BF2E387A
3D8E5821
3FA797AE
3EAA0FB6
3EE0542B
BF83CD40
3F565FD3
BFB84E16
BF7C8F04
BF8EBE57
BF81A29D
C00CF81C
BF93B247
BFDF31F5
3EC6CC21
BD1F762A
3E0B5528
3F9D1290
BF54AE89
3DBCB2FC
3F3BD72F
3EF06AE4
3F201DA9
3ED191A7
3E8D13AC
3EDFAE78
3EF1BAB2
BE35226A
3F4634A5
BD49C5F9
BEA5CC32
3F1DD96F
3F830BDD
3E26C0A6
3FC8B7B2
3F9CBA2C
BF12CC5F
BF47D3CB
3F8D02E4
BFCBE878
3F0C5F3A
3F87E28F
3E7334FE
3FA7B1DA
BFF0D930
BC2B53C0
3F531CCE
BFF88EEC
3E92E4A7
3FDA9E64
BEC70652
3FE24C5C
3EAE91F3
BEBCCDE1
BF23FCB1
BFC933CF
BF08FC8D
3FC91A93
BF9252B1
3F12F9B5
BFA27C89
BEAA103C
BFFA9069
3F287F69
BDD50F49
402633A8
3F9F700B
3EBB5A73
C00BC53E
3E981B3C
BFEA1194
BE8A5058
BF823BDA
40575E41
3F9BECD3
3E526EE0
3F79CC69
3EECD54E
3F1D66E9
BE90A261
402638B9
BF02730E
BF7BC4C6
3ED4B051
BE915AE8
BC94C198
3EFC19D5
BF71772A
BF2A21B9
3E1C59DD
BF321BE1
3F42962C
BF636E75
BF6AB3D1
BF1F467C
3EA051C7
BF8A1334
3EDAB037
3F2A7804
3F305FEC
C01495BA
BF1B56BA
BF034056
401498BC
3E62A6F3
3F488F77
3FA8B30A
BF6A9DD2
3F1FA201
3FE36250
3FCB4518
3F994E63
3DFCBDBF
3F35738B
BFECDC5B
BEBA6751
4004D35A
BE1F7D03
BE06770F
BE0A7F9E
BEFED126
3E7A6A8F
BFA6A745
BE70E9FC
BF91A61F
BFB7A88D
3F967D33
40143BE3
BF5FA47C
BEB70856
BFA489C1
3F780229
