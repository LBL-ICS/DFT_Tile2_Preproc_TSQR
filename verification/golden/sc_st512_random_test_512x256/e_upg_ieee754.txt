BF950E01
3EB79F66
BFA103BE
BF5560D4
BFD2D864
3F5ADB57
BFC6438D
3F3A39A6
3E34B4FA
3CCB7C01
BF6BCD78
3F28BBE2
3FB61D9A
C001DE22
3CDAA231
BFAAF8D1
3EAD2F43
3EDEF315
3F0F706B
3EF99F5D
BF870638
BFD05046
3F5DCB11
BCC3726D
3F9E0F5A
BF871087
BDE18FD6
3F133690
BF77E592
BF11D71E
BE84EF85
3F3ED899
BEEEBFF3
BF935C5B
BEC59205
3F49F6D8
3FF4A822
3F61B160
BDDB6911
BFAF2B52
BF05094A
3EE295F9
BEB9368E
BF295C8A
3FD149A7
3E2A58FC
3F42AACB
3FDB95E2
3F524517
3EA326D2
3FF6788D
3F835A77
BF9AC86D
3EE85561
BFD1DA3F
3DEE3EEA
BE98F35C
BFA52B00
3F6F5593
BF2D06D9
3FCF1132
3F3CCD20
3FAB1085
3FCF925F
3FBD63C8
3F31280A
BDBD1E15
3F38FF21
3E6B230C
BF3BE515
3E689C0A
3F544074
4004FDDB
3CA884D1
BDA96181
BFA87DE7
BE934FBC
BF3B165D
BF2E5E61
BF823471
BEB61AAF
3FFA4334
3F14A9D9
3E61785E
BF29E9D4
3E01E3A2
BE090C18
3F064E82
BDE3A808
BFA71174
3FD46046
3F5D3C17
BE689BAC
BFD9E302
3FEDFBDA
3E6A93BB
BE036A07
BFE30D5A
3FC32051
3EFC2852
BF837D3C
BF955165
BF8E614F
3F709DAC
3F5065E1
BF92F6E6
3FB7C95E
400E0F08
3FCDAF8C
BF391FD4
BF29ED53
3F0BD012
400C3017
3ED268E5
3F80C571
BEE04C1B
3F3E4BFE
BFABE91C
BD590EF2
BFA39F8A
C019E261
3F242A84
BE4DAA67
3F9BC146
C01AB08D
BE3CA611
3CBA8C30
3F77C423
3F977135
BF24377E
3F17C250
C00CACAE
3E1ED8A7
BFA00248
BE65BF90
BF89C68D
BF8D87F9
3ED82888
BCB29A47
3DE69594
3F8AB190
BFACDA70
BF2654C8
3FE7527F
3F886D6E
3F29F64D
BF1B4320
3DE6C496
3EBCF385
3F271012
3F684075
BDCC4B56
BEAB0A79
BDD946A2
400D3079
3F35CC7D
3F759E97
BF59BB9E
BFEB0D00
BFFFFC03
BFF43F6E
3E16A1E8
3F712A46
BDA2D1D4
3E235C65
400602DE
BDEC4D66
3F45F5B4
3D625AAA
BFC70368
3F7213DC
3F461244
40295143
3F4A34B3
BE8157FE
3F8B7E05
BEE93AED
BF8A8A69
BEA6C447
3E64184F
3FB8435D
3B3DEF56
BE3C3F2B
3FB47223
3FEA8EC1
BDFF4147
C02BA9F2
BF1260FF
3F0EF395
BF55453B
3FDDB2EB
BC5E70EB
BE051E4C
3F432F87
3FB44E82
3E563B6A
BEB4156D
3F9BDE55
3F5C26DB
3F3B0CC8
3E2933C3
BE2D35CD
BF37C552
BF891204
401A9B30
3DB5F5F7
BF31D67A
BC97D36A
BFAABF1F
BF10F0F4
3F00CC42
BF70FD28
3FD7D491
BFA7136E
3F00CD1A
BFDC7206
BDEC9CEC
BF0C23E9
3CF397CD
3D96E9E0
3D961C87
BF905B9E
BF8A7CF9
BF49BAF8
3FA25279
BF966DE2
BF55270F
BF144B70
BF999069
BEE725E5
3DF9F3BB
BCD8D9A9
BF05ABE0
BE28AF99
BF7E3D49
3F4567F6
BF1B3083
BF6062A9
BEA05E67
3F2A7FBB
BFA779FF
BF22BA57
BF25074E
BEFA5CF4
3E129E2D
3F3D9093
3E6C892E
BF3B5415
BFE2C4D2
400FC964
3F110C4F
BFA25D7D
3E194CD0
3FAD2DFC
BF43B8EB
BEE203D2
